modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -903,7 +908,7 @@ public void testState() throws Exception {
 
         assertEquals(EXIT_CODE_OK, execute("--state"));
 
-        assertContains(log, testOut.toString(), "Cluster is inactive");
+        assertTrue(patternBuilder.apply(ignite.cluster().state().toString()).matcher(testOut.toString()).find());
```

Comment:
Looks like it can be simplified even more.

Reply:
Done, I changed cluster states to string values in all ‘apply()’ methods in testState().

Reply:
The ```assertClusterState()``` method is used to further simplify the process of checking current cluster state against the '--baseline' and '--state' commands output.

Reply:
Abbreviation plugin has been used to edit the name of the 'logOutput' parameter in the ```assertClusterState()``` method.